src/gc.c

```diff
@@ -609,7 +609,7 @@ size_t gc_run(GarbageCollector* gc)
 {
     LOG_DEBUG("Initiating GC run (gc@%p)", (void*) gc);
     gc_mark(gc);
-    return gc_sweep(gc);
+    return gc_sweep(gc, false);
```

Comment:
See above. Not a fan of `bool` parameters w/o readable `typedefs` or `#defines`.